packages/strapi-plugin-content-manager/admin/src/components/InputFileWithErrors/InputFile.js

```diff
@@ -0,0 +1,64 @@
+import React, { memo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { InputFile as Input } from 'strapi-helper-plugin';
+
+function InputFile({
+  onBlur,
+  onChange,
+  multiple,
+  error,
+  dispatch,
+  name,
+  value,
+  setLabel,
+}) {
+  useEffect(() => {
+    dispatch({
+      type: 'SET_CHECK',
+    });
+  }, [dispatch]);
```

Comment:
The eslint `exhaustive-deps` rules forces you to add dispatch in your dependency array since it is a prop received from your parent. I will let you read [this blog post](https://leewarrick.com/blog/react-use-effect-explained/) for more explanations but here your effect will be run twice.

To avoid it you can do the following:

```
const dispatchRef = useRef();
dispatchRef.current = current;

useEffect(() => {
  dispatchRef.current({ type: 'SET_CHECK' });
}, [dispatchRef]);
```